test/E2ETests/Automation.CLI/CLITestBase.cs

```diff
@@ -20,7 +20,7 @@ public class CLITestBase
         private const string PackagesFolder = "packages";
 
         // This value is automatically updated by "build.ps1" script.
-        private const string TestPlatformCLIPackage = @"Microsoft.TestPlatform.16.9.1";
+        private const string TestPlatformCLIPackage = "Microsoft.TestPlatform.16.9.1";
```

Comment:
Please make sure this update didn't break `build.ps1`.

Reply:
ran this, appears to build

Reply:
There was a small issue with the regular expression on the build.ps1. I fixed it.